Libs/Image/Image.h

```diff
@@ -244,7 +244,7 @@ class Image
   TransformPtr createRigidRegistrationTransform(const Image &target, float isoValue = 0.0, unsigned iterations = 20);
 
   /// creates a vtkPolyData for the given image
-  static vtkSmartPointer<vtkPolyData> getPolyData(const Image& image, PixelType isoValue = 0.0);
+  static Mesh getMesh(const Image& image, PixelType isoValue = 0.0);
```

Comment:
Unless there is a specific reason for this function to be static, let's simply make it a member of Image.

Reply:
Done